modules/Bio/EnsEMBL/Compara/RunnableDB/OrthologQM/GeneOrderConservation.pm

```diff
@@ -172,6 +173,11 @@ sub write_output {
     
     my $goc_scores  = $self->param('goc_scores');
     my $output_file = $self->param('output_file');
+    my $mlss        = $self->param_required('mlss');
```

Comment:
I don't think it is necessary to mark is as required since `$mlss` is used during `fetch_input` (line 71), so if it is not defined, there will be an error raised there.